guide/14-deep-learning/doccano setup guide.ipynb

```diff
@@ -0,0 +1,82 @@
+{
```

Comment:
<blockquote>Run install.bat as administrator. Please be patient, the setup will take ~40 minutes to complete.</blockquote><p> </p><p>Suggestion: install.bat can be mentioned in Italics.</p><p> </p><blockquote>Navigate to doccano_deployment_scriptfolder.</blockquote><p> </p><p>Suggestions:</p><p> </p><p>a) Space to be provided. Folder name can be mentioned in Italics. "Navigate to <em>doccano_deployment_scripts</em> folder".</p><p> </p><p>b) Is it <em>doccano_deployment_scripts or doccano_deployment_script ? </em>Instructions below suggest "<em>doccano_deployment_scripts"</em></p><p> </p><p><em>c) </em>Instructions can be mentioned on where to locate the folder.</p><p> </p><blockquote>Open your browser and go to https:\localhot:8000\</blockquote><p> </p><p>Suggestions:</p><p> </p><p>a) The above instruction does not work (should be http instead of https). Also spelling of ''localhost'' is incorrect. Please change the URL to "<strong>http:\localhost:8000\</strong>".</p><p> </p><p>b) URL can be mentioned in quotes.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/644/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>